pyhoma/client.py

```diff
@@ -20,12 +20,20 @@
 class TahomaClient:
     """ Interface class for the Tahoma API """
 
-    def __init__(self, username: str, password: str, api_url: str = API_URL) -> None:
+    def __init__(
+        self,
+        username: str,
+        password: str,
+        api_url: str = API_URL,
+        session: aiohttp.ClientSession = None,
```

Comment:
Can you just write `session: ClientSession = None,`? and import the right thing? It's not the style we use in the code currently.

Reply:
Should I change it on line 39 as well? Before my modification, it was already written as `aiohttp.ClientSession()`

Reply:
If you can yes.

Reply:
Fixed!